doc/release-notes.md

```diff
@@ -35,17 +35,19 @@ reindex or re-sync the whole chain.
 
 # Notable changes
 
-## Qt Testnet Crash
+## Masternode fix
 
-A crash has been fixed which has only been seen on testnet and only affects QT clients. 
+A problem has been fixed in old quorum data cleanup mechanism that was slowing down Masternode during DKGs causing them to get PoSe scored.
+
+## Testnet Crash
+
+A crash has been fixed which was reported when updating from v19.x to v20.0.0 after v20 activation without re-indexing.
 
 ## Guix Build System Enhancements
-The Guix build system has been enhanced to enable building with custom options when needed.
-This will be used to support custom builds such as for nightly builds with extra debug options.
 
-Additionally, the Guix system will now produce debug symbols for MacOS. 
+The Guix build system has been enhanced with caching and increased timeouts.
```

Comment:
That's for Github CI only.

Reply:
So we shouldn't mention that?

Reply:
Could maybe make it `Other changes` section and say smth like `Implemented improvements in Github CI and build system for macOS. Fixed compilation issues on FreeBSD.`